### PR TITLE
644 Sharing minio presigned URL

### PR DESCRIPTION
**Pull request by longshuicy**
This PR is based off #624.
Grouping the **sharing file** and **sharing visualization** together

image
image

### CODE REVIEW

**Review by ddey2 — APPROVED** [review]
Works fine.

**Review by lmarini — APPROVED** [review]

File: frontend/src/components/sharing/PresignedUrlShareModal.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	return (
+		<Dialog
+			open={presignedUrlShareModalOpen}
+			onClose={() => setPresignedUrlShareModalOpen(false)}
+			fullWidth={true}
+		>
+			<DialogTitle>Share Link</DialogTitle>
+			<DialogContent>
+				<ClowderFootnote>
+					Copy and share the link below. Please note that the link will expire
```
[the remark this answers is not in the document]
Comment (longshuicy) [reply frontend/src/components/sharing/PresignedUrlShareModal.tsx:38]: Ah you are right.. will update it to 1 hour

[the remark this answers is not in the document]
Comment (longshuicy) [reply frontend/src/components/sharing/PresignedUrlShareModal.tsx:38]: actually this is meant for sharing over time ... so maybe i should update the default config to 7 days. Thoughts?

[the remark this answers is not in the document]
Comment (longshuicy) [reply frontend/src/components/sharing/PresignedUrlShareModal.tsx:38]: https://github.com/clowder-framework/clowder2/pull/655/commits/ffb491248bc201b286a0c83ae4ca19bf16f65cd8

**Pull request merged**